src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -368,4 +368,7 @@ Modelers:
   <data name="SummaryDescriptionVaidationError" xml:space="preserve">
     <value>The summary and description values should not be same.</value>
   </data>
+  <data name="LongRunningOperationsWithLongRunningExtension" xml:space="preserve">
+    <value>A long running operation should have 'x-ms-long-running-operation' extension enabled.</value>
```

Comment:
We should probably include in the message what's triggering us to think this is a long-running operation. Would it be more accurate to say something like: "This operation returns 202 status code, which indicates a long running operation, please enable "x-ms-long-running-operation" extension.